export_database.py

```diff
@@ -0,0 +1,120 @@
+from pymongo import MongoClient
+from pymongo import errors
+import bson
+from bson.json_util import dumps, loads
+from os import environ as env
+import sys
+
+old_mongodb_uri = ""
+new_mongodb_uri = ""
+
+def export_db():
+    '''
+        Tries to Exports the whole Database (each and every notice)
+        Saves notices already in target Database in repeated_notices.bson
+        Saves notices failed to be inserted in defaulters.bson
+
+        Can be run multiple times with no fear of multiple entries in case of any confusion
+    '''
+    print("connecting to old DB")
+    mc_old = MongoClient(old_mongodb_uri)
+    print("collecting notices cursor from old database")
+    notices_cursor = mc_old.get_default_database().notices.find()
+    print("connecting to new DB")
+    mc_new = MongoClient(new_mongodb_uri)
+    print("connected to new DB")
+
+    defaulters = []
+    repeated_notices = []
+    for notice in notices_cursor:
+        try:
+            mc_new.get_default_database().notices.insert(notice)
+            print("inserted notice: ", notice)
+        except errors.DuplicateKeyError:
+            print("entry already in database")
+            repeated_notices.append(notice)
+        except Exception as ex:
+            print("error in inserting: ", ex)
+            defaulters.append(notice)
+
+    if(len(defaulters)):
+        print("Defaulters Present!!!")
+    else:
+        print("Yohoo!! No Defaulters")
+
+
+    print("Saving defaulters and repeated notices...")
+    open("defaulters.bson", "w").write(dumps(defaulters))
+    open("repeated_notices.bson","w").write(dumps(repeated_notices))
+
+    print("Export Complete!")
+    mc_old.close()
+    mc_new.close()
+
+def insert_notice(notice, mc_new):
+    '''
+        Tries to insert a notice in default database of MongoClient passed as second argument
+        Returns 0 if insert successful
+        Returns 2 if entry already in database
+        Returns 1 if some other error occurs
+    '''
+    try:
+        mc_new.get_default_database().notices.insert(notice)
+        print("inserted specific notice: ", notice)
+        return 0
+    except errors.DuplicateKeyError:
+        print("specific entry already in database")
+        return 2
+    except errors.ConnectionFailure:
+        print("Error while connecting to DB")
+        return 1
+    except Exception as ex:
+        print("error in specific inserting: ", ex)
+        return 1
+
+def insert_from_file(filename, further_defaulter_filename = "further_defaulters.bson", further_repeated_filename = "further_repeated.bson"):
```

Comment:
Can you please help me understand the usage of this function in our purpose of backing up from one database to another?